server/src/cli.rs

```diff
@@ -4,7 +4,9 @@ use clap::{ArgGroup, Args, Parser, Subcommand};
 
 #[derive(Subcommand, Debug, Clone)]
 pub enum EdgeMode {
+    /// Run in offline mode
```

Comment:
I don't think these comments add much, they seem to be describing what the enum variant does. If the enum is unclear, I think we should rather just rename it

Reply:
Fair enough!